api/views.py

```diff
@@ -38,7 +57,18 @@ def analyze_address(request):
 
 @csrf_exempt
 def get_streets(request):
-    with Path('api/data/streets.json', encoding='utf-8').open() as f:
-        street_data = json.load(f)
-    street_names = street_data.get("t_rechov_values", [])
-    return JsonResponse({"streets": street_names})
+    try:
+        data_dir = Path(__file__).resolve().parent / 'data'
+        streets_path = data_dir / 'streets.json'
+        with streets_path.open(encoding='utf-8') as f:
+            street_data = json.load(f)
```

Comment:
IIUC, this code loads the file on every request? You usually want to cache these kind of things. You could e.g. check the modification time on the file and read it only if it changed since last time.